### DISCUSSION TITLE
App access problem

### DISCUSSION THREAD

**Discussion by nithin-techforce** [directus/directus#3569] [Archive]
issue:
when creating the roles and assigning to users through directus application ,user is able to get the app access and able to login with the credentials but when the same thing is repliacted with the api calls manually although i am providing app_access as true when creating role and  assigned user to the role created by api call ,when user logging in its restricting the login by saying no app access

payload for creating role:
{
"name":"role-name",
"app_access":true,
"admin_access":false
}

status :200 on response
![Screenshot (101)](https://user-images.githubusercontent.com/73542860/103521538-efce3500-4e9e-11eb-88c9-f537049ff075.png)

**Comment by benhaynes** [comment]
Did you manually change any of the system permissions? You can try going to the bottom of permissions and clicking "reset" to see if that helps.
